example/main.dart

```diff
@@ -112,7 +112,7 @@ Future<Null> captureCompleteExampleEvent(SentryClient client) async {
             timezone: 'America/Toronto',
           )));
 
-  final response = await client.capture(event: event);
+  final response = await client.captureEvent(event: event);
```

Comment:
I was also thinking of moving to positional arguments here. The call would be:

`captureEvent(event)`; in that case.

With regards to the `StackFrameFilter`, I'd like to remove that as an argument.
It will be provided via the `SentryOptions` object, via the constructor of the client.

I hope these change make sense, I hope to open PRs for that asap if you agree.